### ISSUE TITLE
[TS 3.4] support readonly operator

### ISSUE THREAD

**Issue by ikatyang** [typescript-eslint/typescript-eslint#358]
[TS 3.4 RC release note](https://github.com/Microsoft/TypeScript/releases/tag/v3.4-rc)

**What code were you trying to parse?**

```ts
declare const array: readonly number[];
//                   ^^^^^^^^^^^^^^^^^
```

**What did you expect to happen?**
```json
{
  "type": "TSTypeOperator",
  "operator": "readonly",
  "typeAnnotation": {
    "type": "TSArrayType",
    "elementType": {
      "type": "TSNumberKeyword"
    }
  }
}
```

**What actually happened?**
```diff
 {
   "type": "TSTypeOperator",
-  "operator": "readonly",
+  "operator": undefined,
   "typeAnnotation": {
     "type": "TSArrayType",
     "elementType": {
       "type": "TSNumberKeyword"
     }
   }
 }
```

**Versions**

| package                                | version |
| -------------------------------------- | ------- |
| `@typescript-eslint/typescript-estree` | `1.4.3-alpha.5` |
| `TypeScript`                           | `3.4.0-rc` |

---

It should be just a matter to add `ReadonlyKeyword` to this list:

https://github.com/typescript-eslint/typescript-eslint/blob/3b28cacd49a1bc61ba0c3f8f76096f6902b92848/packages/typescript-estree/src/node-utils.ts#L28-L29

---

Original post: prettier/prettier#5977

**Comment by JamesHenry** [comment]
Thanks @ikatyang!

This has been released in `1.6.0`
